src/components/NewRuntimeModal.js

```diff
@@ -834,6 +865,7 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
         h(TitleBar, {
           style: styles.titleBar,
           title: h(WarningTitle, ['Delete environment options']),
+          hideCloseButton: isAnalysisMode,
```

Comment:
the close button is hidden on all inner modal title bars (aka if accessed in analysisMode)